I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 27, in script
    $ cash += 10
  File "game/script.rpy", line 27, in <module>
    $ cash += 10
NameError: name 'cash' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 27, in script
    $ cash += 10
  File "C:\Users\user\Desktop\renpy-7.3.5-sdk\renpy\ast.py", line 914, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\user\Desktop\renpy-7.3.5-sdk\renpy\python.py", line 2028, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 27, in <module>
    $ cash += 10
NameError: name 'cash' is not defined

Windows-8-6.2.9200
Ren'Py 7.3.5.606
TimeNMoney 1.0
Sun Oct 04 11:52:33 2020
